Lab7.py

```diff
@@ -0,0 +1,48 @@
+
+class Stadium:
+    def __init__(self, name="empty", capacity=0, current_attendance=0, home_team="empty", away_team="empty"):
+        self.name = name
+        self.capacity = capacity
+        self.current_attendance = current_attendance
+        self.home_team = home_team
+        self.away_team = away_team
+
+    instance = None
+
+    def add_attendance(self, count):
+        if self.capacity >= self.current_attendance + count:
+            self.current_attendance += count
+
+    def decrease_attendance(self):
+        DECREASE = 100
```

Comment:
this is a variable declaration, and variable name does not correspond to convention. Make It as a constant